drivers/vulkan/vulkan_context.cpp

```diff
@@ -1727,7 +1711,7 @@ Error VulkanContext::_update_swap_chain(Window *window) {
 		/*imageSharingMode*/ VK_SHARING_MODE_EXCLUSIVE,
 		/*queueFamilyIndexCount*/ 0,
 		/*pQueueFamilyIndices*/ nullptr,
-		/*preTransform*/ (VkSurfaceTransformFlagBitsKHR)preTransform,
+		/*preTransform*/ static_cast<VkSurfaceTransformFlagBitsKHR>(preTransform),
```

Comment:
I don't know what's the consensus on casting style in Godot. I'd say in general we don't have issues with C-style casts where there's no ambiguity.